src/elements/Svg.tsx

```diff
@@ -45,6 +45,36 @@ export interface SvgProps extends GProps, ViewProps, HitSlop {
   title?: string;
 }
 
+export type DataUrlOptions = JpegOptions | PngOptions;
+
+interface JpegOptions {
+  format: 'jpeg';
+  quality?: number;
+  width?: NumberProp;
+  height?: NumberProp;
+}
+
+interface PngOptions {
+  format: 'png';
+  width?: NumberProp;
+  height?: NumberProp;
+}
```

Comment:
We should probably require width, when height is passed (and vice versa)

Reply:
Done